rep-0149.rst

```diff
@@ -992,6 +993,18 @@ conditions which use those to limit the scope of dependencies.
 If no environment has been sourced some tools might require that the necessary
 information is being specified explicitly when being invoked.
 
+New ROS package
+'''''''''''''''
+
+In ROS 1 the environment variable `ROS_DISTRO` is being set in the `roslib`
+package which also defines other environment variables like `ROS_PACKAGE_PATH`.
+In ROS 2 the environment variable `ROS_DISTRO` doesn't exist at the moment.
+Also neither ROS version has an environment variable `ROS_VERSION` at the
+moment.
+
+In order to make these environment variables available in a package which is
+as separate as possible from other package content and the same in both ROS
+versions a new ROS package named `ros_environment` is being introduced.
```

Comment:
It seems weird to talk about this in the future tense for this REP (since the REP will long out last this temporary state where we plan to do but haven't yet). It might be better to put hard dates and milestones in here instead, like "until Feb 2018" or "until ROS Melodic" to describe the previous state. If need be you could leave this pr open until it is done or make a follow up pr to put in firm dates once it is done.

Reply:
I rephrased it in 8f6ec220a577556f26e2c7978a66932bc7d8ff75:

> A new ROS package named `ros_environment` which has minimal dependencies will
be available in both ROS versions and providing the new environment variables
as well as some of the existing environment variables.

Reply:
lgtm